components/buttons/ChapterSuggestion.js

```diff
@@ -9,19 +10,21 @@ export default function ChapterSuggestion({
   className,
 }) {
   return (
-    <a
-      href={href}
-      target={target}
-      rel="noopener noreferrer"
-      title={description}
-      className={`${className} flex flex-col mt-4 p-6 max-w-lg rounded shadow-md border border-gray-200 text-gray-700 transition-all duration-300 ease-in-out hover:shadow-lg hover:border-gray-300`}
-    >
-      <h5 className="text-lg font-medium font-sans antialiased mb-2">{title}</h5>
-      <p className="flex-1 mb-2 font-normal font-sans antialiased">{description}</p>
-      <p className="text-primary-600 font-medium font-sans antialiased">
-        {linkText}
-        <IconArrowRight className="inline-block h-4" />
-      </p>
-    </a>
+    <Link href={href}>
+      <a
+        href={href}
```

Comment:
Duplicated `href`.

Reply:
Also, I don't see where suggestions switch me when I click on the suggestion item -> more info here https://github.com/asyncapi/website/pull/585#issuecomment-1033774166

Reply:
I also see that we have duplicated `href` in other places. NextJS handles that under the hood, so `href` should be only in the `Link` component:

https://nextjs.org/docs/api-reference/next/link

(check examples)

Reply:
No problem :) 

> So sorry, I accidentally missed this href to be removed from a tag. I will correct it in the next commit. But to inform you, this ChapterSuggestions Component is not been used anywhere now.

Yeah, sorry, I mean that new component to show suggestions in docs :)